app/topics/topic-management/add-existing-subtopic/add-existing-subtopic.component.ts

```diff
@@ -36,12 +36,26 @@ export class AddExistingSubtopicComponent {
     this.showChange.emit(this.show);
   }
 
+  private filterTopics(topics: any) {
```

Comment:
there i no need for the parameter topics, since you wrote it already in a class variable (`this.searchResults`)